### PR TITLE
Fix analytics hero section

### PR DESCRIPTION
**Pull request by Arth101**
Simple hero section with USP. It works

Desktop:
image

Mobile:
image

### CODE REVIEW

File: _includes/fix-analytics-hero.html [hunk 1]
```diff
@@ -1,39 +1,27 @@
...
-        </h3>
-        <a href="">Read more</a>
-      </div>
+  <h1 class="heading heading--big text--centered">
+    The <em class="heading__emphasis text--fancy text__fastest--fancy">fastest</em>
+    and <em class="heading__emphasis text--fancy text__cheapest--fancy">cheapest</em>
+    way to stay GDPR-compliant
+  </h1>
+  <div class="card selling-propositions">
+    <div class="selling-propositions__item d--f fd--c jc--c">
```
Comment (hypesystem) [inline _includes/fix-analytics-hero.html]: plsfix display + fd + jc classes to something readable :sob:

File: _sass/fix-analytics/hero-section.scss [hunk 2]
```diff
@@ -7,3 +7,47 @@
     padding-bottom: 170px;
   }
 }
+
+.selling-propositions {
```
Comment (hypesystem) [inline _sass/fix-analytics/hero-section.scss]: Could we make this a more general element instead?

I think a side-by-side list of things in a card could be generally described probably

**Review by hypesystem — APPROVED** [review]
Looks good! Only some notes on naming, all the code is otherwise v gud

File: _sass/fix-analytics/hero-section.scss (hunk 2 above)
Comment (Arth101) [reply _sass/fix-analytics/hero-section.scss]: I have created a `card__usp` element now :)

**Pull request merged**